SanaraV3/Module/Entertainment/JapaneseModule.cs

```diff
@@ -213,6 +214,70 @@ public async Task LightNovelAsync([Remainder] string name)
             await PostAnimeEmbedAsync(JapaneseMedia.LIGHT_NOVEL, await SearchMediaAsync(JapaneseMedia.LIGHT_NOVEL, name));
         }
 
+        [Command("drama", RunMode = RunMode.Async)]
```

Comment:
All others commands start with a capital letter on their Command attribute
```suggestion
        [Command("Drama", RunMode = RunMode.Async)]
```